components/infobox/wikis/starcraft2/infobox_team_custom.lua

```diff
@@ -141,23 +126,23 @@ function CustomTeam:createWidgetInjector()
 end
 
 function CustomTeam:createBottomContent()
-	if doStore then
-		return tostring(Matches._get_ongoing({})) ..
-			tostring(Matches._get_upcoming({})) ..
-			tostring(Matches._get_recent({}))
+	if _doStore then
+		return tostring(Matches._get_ongoing{}) ..
+			tostring(Matches._get_upcoming{}) ..
+			tostring(Matches._get_recent{})
```

Comment:
Why does an array need to be inserted?

Reply:
to avoid nil
i can adjust the other module^^